context/app/static/js/pages/CellsAPIDemo/DatasetsSelectedByGene.jsx

```diff
@@ -38,29 +39,36 @@ function DatasetsSelectedByGene(props) {
     const { name } = target;
     const setFields = {
       geneName: setGeneName,
-      minGeneExpression: setMinGeneExpression,
-      minCellPercentage: setMinCellPercentage,
     };
     setFields[name](event.target.value);
   }
 
   return (
     <Paper>
       <TextField label="gene name" value={geneName} name="geneName" variant="outlined" onChange={handleChange} />
-      <TextField
-        label="min gene expression"
+
+      <p id="min-gene-expression-label">Minimun gene expression</p>
```

Comment:
Use label instead of p?

Reply:
I think MUI has a label component for inputs/forms.

Reply:
Sure, I'll use that instead ... but John is going to have a lot of these to fix, in any event.